impeller/tessellator/tessellator_unittests.cc

```diff
@@ -83,26 +83,34 @@ TEST(TessellatorTest, TessellatorBuilderReturnsCorrectResultStatus) {
 TEST(TessellatorTest, TessellateConvex) {
   {
     Tessellator t;
+    std::vector<Point> points;
+    std::vector<uint16_t> indices;
     // Sanity check simple rectangle.
-    auto pts = t.TessellateConvex(
-        PathBuilder{}.AddRect(Rect::MakeLTRB(0, 0, 10, 10)).TakePath(), 1.0);
-
-    std::vector<Point> expected = {{0, 0}, {10, 0}, {0, 10}, {10, 10}};
-    EXPECT_EQ(pts, expected);
+    t.TessellateConvexInternal(
+        PathBuilder{}.AddRect(Rect::MakeLTRB(0, 0, 10, 10)).TakePath(), points,
+        indices, 1.0);
+
+    std::vector<Point> expected = {{10, 0}, {10, 10}, {0, 10}, {0, 0}};
```

Comment:
There order of these changes because we hit the "first" point in the contour on close now.